apps/jobs/constants/values.py

```diff
@@ -1,11 +1,21 @@
-"""This file contains constant which are also used in other parts of the jobs API code, 
+"""This file contains constant which are also used in other parts of the jobs API code,
```

Comment:
file name should be constants.py
if config 
configs.py
if msg or response response.py